packer/http/opensuse-leap-42.2/autoinst.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+
+<!-- http://doc.opensuse.org/projects/autoyast/configuration.html -->
+
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
```

Comment:
It's interesting that the second URL leads to a 404. I hope the generator is up-to-date